packages/x-components/src/plugins/__tests__/x-plugin.spec.ts

```diff
@@ -440,9 +472,21 @@ describe('testing X Plugin', () => {
       component.vm.$x.emit('UserIsTypingAQuery', 'chinchulines');
       component.vm.$x.emit('UserIsTypingAQuery', '');
 
-      await localVue.nextTick();
+      await waitNextTick();
 
       expect(searchBoxQueryChangedSubscriber).toHaveBeenCalledTimes(0);
     });
   });
 });
+
```

Comment:
Can we extract this method to a `utils` folder an avoid the code in these two tests duplicate?

```javascript
import { default as Vue, VueConstructor } from 'vue';
/**
 * Waits for Vue's reactivity to update getters and watchers, and flushes the pending emitters.
 *
 * @remarks It needs `jest.useFakeTimers()` to have been called to wait for the emitters.
 * @returns A promise that resolves after the reactivity has been updated and the pending emitters
 * have been run.
 */
export async function waitNextTick(localVue: VueConstructor<Vue>): Promise<void> {
  await localVue.nextTick();
  jest.runAllTimers();
}
```